.github/workflows/validate-bicep.yaml

```diff
@@ -44,6 +44,14 @@ jobs:
           bicep --version
       - name: Check out repo
         uses: actions/checkout@v4
+      - name: Install jq
+        run: sudo apt-get install -y jq
+      - name: Modify bicepconfig.json
+        run: |
+          # Add credential preference to the bicepconfig.json file
+          jq '. + {"cloud": {"credentialPrecedence": ["Environment"]}}' bicepconfig.json > tmp.json && mv tmp.json bicepconfig.json
+      - name: Print updated bicepconfig.json
+        run: cat bicepconfig.json
```

Comment:
Do we need this in production?

Reply:
No, this was the initial workaround since there were authentication issues with the ACR even though we have anonymous pulls. The issue is that when we publish to ACR, we have to authenticate so only have "Environment" authentication removed any Azure CLI auth. Ideally, we can remove this everywhere in the repo once the bug is fixed: https://github.com/radius-project/radius/issues/7804